htdocs/install/mysql/migration/16.0.0-17.0.0.sql

```diff
@@ -84,3 +84,8 @@ ALTER TABLE llx_societe ADD last_main_doc VARCHAR(255) NULL AFTER model_pdf;
 ALTER TABLE llx_ticket ADD COLUMN ip varchar(250);
 
 ALTER TABLE llx_ticket ADD email_date datetime after email_msgid;
+
+
+ALTER TABLE llx_mrp_mo ADD COLUMN predicted_cost double(24, 8) DEFAULT 0 AFTER qty;
+
+ALTER TABLE llx_mrp_mo ADD COLUMN real_cost double(24, 8) DEFAULT 0 AFTER qty;
```

Comment:
I suppose that
real cost is the "observed price" once production is finished ?

But what is predicted_cost ? Is it an estimation done by end user or a calculation ?

Reply:
it's the qty with the flag tocosume in mrp_production

Reply:
and it's a  calculation